daemon/cluster/swarm.go

```diff
@@ -25,19 +25,20 @@ import (
 func (c *Cluster) Init(req types.InitRequest) (string, error) {
 	c.controlMutex.Lock()
 	defer c.controlMutex.Unlock()
-	c.mu.Lock()
```

Comment:
`c.nr` only changes when `c.controlMutex` is being held so there is no need to check for that. It is only for synchronization to the functions that take `c.mu.RLock()`. Although using local var might help readability.